README.md

```diff
@@ -5,17 +5,17 @@
 [![coverage](https://raw.githubusercontent.com/hirosystems/stacks-wallet-web/gh-pages/badge.svg)](https://hirosystems.github.io/stacks-wallet-web/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-Hiro Wallet is a web extension for managing your digital assets, and connecting to apps built on the Stacks blockchain.
+Hiro Wallet is the most popular and trusted wallet for apps built on Bitcoin. Connect to apps and manage assets secured by Bitcoin and BTC L2s with the most popular, battle-tested wallet for the Stacks blockchain.
```

Comment:
Isn't it better to say, even if repetitive, Bitcoin L2. BTC refers more specifically to units of the currency itself. 

We shouldn't repeat the popular twice.